generators/server/templates/build.gradle.ejs

```diff
@@ -250,11 +250,9 @@ configurations {
             // Inherited version from Spring Boot can't be used because of regressions:
             // To be removed as soon as spring-boot use the same version
             force 'org.liquibase:liquibase-core:<%= LIQUIBASE_VERSION %>'
-<%_ if (devDatabaseTypeH2Any) { _%>
+<%_ if (devDatabaseTypeH2Any && reactive) { _%>
```

Comment:
No need for a driver for imperative?

Reply:
It’s overriding the version for reactive only